src/cli/main.cpp

```diff
@@ -29,8 +36,10 @@ int main(int argc, char* argv[]) {
 
   int result = 0;
 
-  std::vector<std::string> args(argv, argv + argc);
-  // erase the first element
+  std::vector<std::string> args(argv, std::next(argv, static_cast<std::ptrdiff_t>(argc)));
+  std::for_each(args.begin(), args.end(), [](auto& s) { openstudio::ascii_trim(s); });
+  // erase the first element, which is the name of the program
+  const std::string programName = std::move(args.front());
```

Comment:
i'm going to be massaging the arguments in a bit, so convert to a vector. CLI11_PARSE(argc, argv) does that too, so this isn't less efficient. More on this in a bit below